test/src/xquery/xquery3/arrowop.xql

```diff
@@ -0,0 +1,47 @@
+xquery version "3.0";
+
+module namespace ao="http://exist-db.org/xquery/test/arrowop";
+
+declare namespace test="http://exist-db.org/xquery/xqsuite";
+
+declare
+    %test:assertEquals("Hello world")
+function ao:func-by-name1() {
+    "Hello" => concat(" ", "world")
+};
+
+declare
+    %test:assertEquals(3)
+function ao:func-by-name2() {
+    ('A', 'B', 'C') => count()
+};
+
+declare
+    %test:assertEquals(3)
+function ao:func-inline() {
+    ('A', 'B', 'C') => (function( $sequence) { count( $sequence)})()
+};
+
+declare
+    %test:assertEquals(11)
+function ao:func-in-sequence1() {
+    "Hello" => concat(" world") => string-length()
+};
+
+declare
+    %test:assertEquals("HELLO", "WORLD")
+function ao:func-in-sequence2() {
+    "hello world"=>upper-case()=>normalize-unicode()=>tokenize("\s+")
```

Comment:
The test result does not show that `normalize-unicode` was or was not called. Perhaps a better test case would be:
```xquery
declare
    %test:assertEquals("123", "abc", "dl")
function ao:func-in-sequence2() {
    "123 abc ㎗" => upper-case() => normalize-unicode() => tokenize("\s+")
}
````